Penumbra/Import/Models/Export/MeshExporter.cs

```diff
@@ -335,28 +343,28 @@ private Type GetGeometryType(IReadOnlyDictionary<MdlFile.VertexUsage, MdlFile.Ve
     }
 
     /// <summary> Build a geometry vertex from a vertex's attributes. </summary>
-    private IVertexGeometry BuildVertexGeometry(IReadOnlyDictionary<MdlFile.VertexUsage, object> attributes)
+    private IVertexGeometry BuildVertexGeometry(IReadOnlyDictionary<MdlFile.VertexUsage, List<object>> attributes)
     {
         if (_geometryType == typeof(VertexPosition))
             return new VertexPosition(
-                ToVector3(attributes[MdlFile.VertexUsage.Position])
+                ToVector3(attributes[MdlFile.VertexUsage.Position].First())

```

Comment:
(ditto on most other `.First()` calls)

these are encoding an assumption that these vertex usages will only have one usageindex (akin to the prior existing assumption) - given we're now handling uv with multiple indices, would be good to fail-fast if this assumption is broken. perhaps just an assertion that the arrays are length 1 (as a helper at least).

if there are existing models with >1 usages for an attribute (other than uv), we should probably intentionally not-handle it, rather than implicitly (or handle it? idk lmao)